src/miral/display_configuration_option.cpp

```diff
@@ -105,15 +110,42 @@ void miral::display_configuration_options(mir::Server& server)
             auto const options = server.get_options();
             auto display_layout = options->get<std::string>(display_config_opt);
             auto with_alpha = options->get<std::string>(display_alpha_opt) == display_alpha_on;
+            auto const scale_str = options->get<std::string>(display_scale_opt);
+
+            double scale{0};
+            static double const scale_min = 0.01;
```

Comment:
I think we can trust the compiler to optimise these out without explicitly marking them `static`, but I also don't think it'll hurt to do so…